CHANGELOG.md

```diff
@@ -4,6 +4,11 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [10.6.1]
```

Comment:
It would be `10.6.0` for a minor release, but technically I think this is a breaking change for a job type that's already available in edc-prod, so technically it should be a major release. But I'm open to push-back since I don't think we've really advertised this job type yet?